payments/models.py

```diff
@@ -1054,6 +1054,14 @@ def __str__(self):
 
 
 class AbstractOffer(UUIDModel, TimeStampedModel):
+    offer_number = models.CharField(
```

Comment:
offer_number could be added to list_display and search_fields in BerthSwitchOfferAdmin